doc/contributing/releases.md

```diff
@@ -311,6 +311,9 @@ $ git checkout -b v1.2.3-proposal upstream/v1.x
 git cherry-pick  ...  # cherry-pick nodejs-private PR commits directly into the proposal
 ```
 
+Remember to flag the CVE fixes as `notable-change` in the nodejs-private so it
+will be automatically included in the CHANGELOG "Notable Changes" section.
```

Comment:
```suggestion
Be sure to label the CVE fixes as notable-change in the nodejs-private repository.
This will ensure they are automatically included in the "Notable Changes" section of the CHANGELOG.
```

Reply:
I would drop "automatically"
```suggestion
Be sure to label the CVE fixes as `notable-change` in the nodejs-private repository.
This will ensure they are included in the "Notable Changes" section of the CHANGELOG.
```